process/charter.html

```diff
@@ -82,262 +82,77 @@ <h4>Tools and templates</h4>
+	
 <p>The Process Document includes a section on <a
 href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">Working Group
-and Interest Group Charter Development</a>, which includes a
-requirement to alert the Advisory Committee when work begins on a
-charter for a new group. See also <a href="/2002/05/rec-tips">tips for
-getting to Recommendation faster</a> and <a
-href="/TR/qa-handbook/">The QA Handbook</a>, which discusses process
-and operational aspects of certain quality assurance practices of
-W3C's Working Groups,</p>
-
-<p>For existing groups, the Chair and Team Contact should work with
-the WG to create a new charter and develop rationale as to why the
-change is necessary. This reason need not be overly formal. For
-instance, it may be no more complex than, "we need three more
-months to complete our deliverable commitments and are fairly
-certain we can complete them within those three months." If the
-Working Group or the Chair proposes changes to the charter, they
-should send them to the Team Contact, who should see that the
-request is reviewed by the Project and Strategy Leads in a timely manner.</p>
-
-<p><strong>Note:</strong> Team Contacts should take caution when
-creating a new charter by copying an old one, especially in light of
-adoption of the W3C Patent Policy. Please review the <a
+and Interest Group Charter Development</a>, which includes a <a
 href="https://www.w3.org/Consortium/Process/#WGCharter">list
-of charter requirements</a> to ensure that your charter is complete
-(e.g., it includes a duration for the group).</p>
+	of charter requirements</a>.</p>
 
-<p class=timeline><em>Timeline:</em> For existing groups, this takes no time. For new groups, it may take one month or more for W3M
-  to start dedicating resources.</p>
+<p>&nbsp;</p>
 
-<h2 id="advance">2. Advance Notice</h2>
+<h2 id="advance">1. New Groups: Advance Notice</h2>
 
 <!-- Resolved at W3M 11 July 2007 -->
 
-<p>W3C Process (<a
-href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">section
-"Working Group and Interest Group Charter Development"</a>) requires that the Team notify the Advisory Committee when a
+<p><a
+href="https://www.w3.org/Consortium/Process/#WGCharterDevelopment">W3C Process</a> requires Team notify the Advisory Committee when a
 charter for a <strong>new</strong> Working Group or Interest Group is
 in development. The exact timing of the advance notice may vary from
 charter to charter. In practice, if the advance notice would precede
 the formal call for review by only a short delay, we skip the advance
-notice; see the <a href="https://www.w3.org/new-doc-from-template?location=%2FTeam%2F&amp;template=%2Fafs%2Fw3.org%2Fpub%2FWWW%2FTeam%2FTemplates%2Fadv-charter.html&amp;submit=Continue...">Advance Notice for Charter template</a></p>
+notice.</p>
 
-<p>The Team should consult <a href="/2005/09/adv-notice.html">How to
+<p>Consult <a href="/2005/09/adv-notice.html">How to
```

Comment:
/2005/09/adv-notice.html is a team-only document, while /Guide is public, with the audience for each page varies. So, unless we intent to change the visibility of adv-notice, "The Team should consult" remains clearer imho.

Reply:
makes sense to me.